Dockerd travis test.

Aniketh01 opened the pull request.

Aniketh01 committed:
travis ci test for dockerd

Aniketh01 committed:
update the yml file

thehajime committed:
Update docker-dockerd-test.sh

thehajime committed:
Update docker-dockerd-test.sh

thehajime committed:
Update docker-dockerd-test.sh

thehajime committed:
Update docker-dockerd-test.sh

thehajime committed:
Update docker-dockerd-test.sh

thehajime committed:
Update docker-dockerd-test.sh

thehajime committed:
Update docker-dockerd-test.sh

thehajime committed:
Update docker-dockerd-test.sh

thehajime committed:
Update docker-dockerd-test.sh

thehajime committed:
Update .travis.yml

thehajime committed:
Update docker-dockerd-test.sh

thehajime committed:
Update docker-dockerd-test.sh

thehajime committed:
Update docker-dockerd-test.sh

thehajime commented:
I managed to pass the test with modified PR: 

please squash all the commits into a single one (even .yml and shell script are in the same commit).

thehajime committed:
spawn containerd from dockerd

https://github.com/ukontainer/dockerd-darwin/commit/d4299e2987b454892342411f3f458e017a27772a

thehajime committed:
spawn containerd take2

thehajime commented:
Committed. 0d7dd1d44b1d716e0d360e58f3140cec95fd46b8

Thanks !
